### PR TITLE
Chapter 8: Basic login

### PR DESCRIPTION
**Pull request by stephanie1997-kek**

### CODE REVIEW

**Comment by stephanie1997-kek** [comment]
ready

File: app/assets/javascripts/sessions.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (saobv-2223) [inline app/assets/javascripts/sessions.coffee]: remove file that never used

File: app/assets/stylesheets/sessions.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (saobv-2223) [inline app/assets/stylesheets/sessions.scss]: remove file that never used

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -12,6 +12,7 @@ def create
     @user = User.new user_params
 
     if @user.save
+      log_in@user
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]:
```suggestion
      log_in @user
```

File: app/views/sessions/_newform.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<%= form_for :session, url: login_path do |f| %>
+  <%= f.label :email %>
+  <%= f.email_field :email, class: "form-control" %>
+  <%= f.label :password %>
+  <%= f.password_field :password, class: "form-control" %>
+	<%= f.submit t(".login"), class: "btn btn-primary" %>
```
Comment (saobv-2223) [inline app/views/sessions/_newform.html.erb]:
```suggestion
  <%= f.submit t(".login"), class: "btn btn-primary" %>
```

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (stephanie1997-kek) [reply app/controllers/users_controller.rb]: done

File: app/assets/javascripts/sessions.coffee (hunk 1 above)
Comment (stephanie1997-kek) [reply app/assets/javascripts/sessions.coffee]: done

File: app/assets/stylesheets/sessions.scss (hunk 2 above)
Comment (stephanie1997-kek) [reply app/assets/stylesheets/sessions.scss]: done

File: app/views/sessions/_newform.html.erb (hunk 4 above)
Comment (stephanie1997-kek) [reply app/views/sessions/_newform.html.erb]: done

**Comment by stephanie1997-kek** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
merge

**Pull request merged**